src/Plugin.php

```diff
@@ -198,6 +201,48 @@ public static function getBasePath() {
     return dirname(__DIR__);
   }
 
+  /**
+   * Replaces default version to Git reference in script URL.
+   *
+   * @return string
+   *   The tag to print with the version updated.
+   */
+  public static function script_loader_tag($tag, $handle, $src) {
```

Comment:
Since core-standards is a collection of many different adjustments and features, we should strive for a clear code organization of features, ideally one feature or topic per class.  Can we move the callbacks (not their registration) into a new class `Asset`?